articles/azure-functions/functions-infrastructure-as-code.md

```diff
@@ -179,124 +179,169 @@ For a sample Azure Resource Manager template, see [Function app on Consumption p
 
 A Consumption plan doesn't need to be defined. One will automatically be created or selected on a per-region basis when you create the function app resource itself.
 
-The Consumption plan is a special type of "serverfarm" resource. For Windows, you can specify it by using the `Dynamic` value for the `computeMode` and `sku` properties:
+The Consumption plan is a special type of "serverfarm" resource. You can specify it by using the `Dynamic` value for the `computeMode` and `sku` properties as following:
+
+# [Windows](#tab/Windows)
 
 ```json
 {
-   "type":"Microsoft.Web/serverfarms",
-   "apiVersion":"2016-09-01",
-   "name":"[variables('hostingPlanName')]",
-   "location":"[resourceGroup().location]",
-   "properties":{
-      "name":"[variables('hostingPlanName')]",
-      "computeMode":"Dynamic"
-   },
-   "sku":{
-      "name":"Y1",
-      "tier":"Dynamic",
-      "size":"Y1",
-      "family":"Y",
-      "capacity":0
-   }
+  "type": "Microsoft.Web/serverfarms",
+  "apiVersion": "2021-02-01",
+  "name": "[variables('hostingPlanName')]",
+  "location": "[parameters('location')]",
+  "sku": {
+    "name": "Y1",
+    "tier": "Dynamic",
+    "size": "Y1",
+    "family": "Y",
+    "capacity":0
+  },
+  "properties": {
+    "name":"[variables('hostingPlanName')]",
+    "computeMode": "Dynamic"
+  }
 }
 ```
 
-> [!NOTE]
-> The Consumption plan cannot be explicitly defined for Linux. It will be created automatically.
+# [Linux](#tab/Linux)
 
-If you do explicitly define your Consumption plan, you will need to set the `serverFarmId` property on the app so that it points to the resource ID of the plan. You should ensure that the function app has a `dependsOn` setting for the plan as well.
+To run your app on Linux, you must also set property `"reserved": true` for the serverfarms resource:
+
+```json
+{
+  "type": "Microsoft.Web/serverfarms",
+  "apiVersion": "2021-02-01",
+  "name": "[variables('hostingPlanName')]",
+  "location": "[parameters('location')]",
+  "sku": {
+    "name": "Y1",
+    "tier": "Dynamic",
+    "size": "Y1",
+    "family": "Y",
+    "capacity":0
+  },
+  "properties": {
+    "name":"[variables('hostingPlanName')]",
+    "computeMode": "Dynamic",
+    "reserved": true
+  }
+}
+```
+---
 
 ### Create a function app
 
+If you do explicitly define your Consumption plan, you will need to set the `serverFarmId` property on the app so that it points to the resource ID of the plan. You should ensure that the function app has a `dependsOn` setting for the plan as well.
```

Comment:
```suggestion
When you define your Consumption plan, you must set the `serverFarmId` property on the app so that it points to the resource ID of the plan. Make sure that the function app has a `dependsOn` setting that also references the plan.
```